TROUBLESHOOTING.md

```diff
@@ -190,3 +191,23 @@ ENOGIT
 ### Related Issues
 
 * [#11095](https://github.com/npm/npm/issues/11095)
+
+## Cannot Read property
+If your install fails and it says something like `Cannot read property 'path' of null`, then it is most likely a network issue in disguise.
```

Comment:
I would tell people to pop open their error logs and look for the error details near the bottom of the log, and compare them with more expanded examples. Both of the examples seem to be referring to `ETIMEDOUT` errors -- can we help clarify that in here?